### ISSUE TITLE
Attention on how to use container class

### ISSUE THREAD

**Issue by bahaa-ka** [bahaa-ka/proco-web-page#1]
i noticed that you replicate the use of container class for each section. yes it will work but its not a good practice especially in this case 
usually in most cases when using bootstrap grid system you will have this structure:
```
<body>
  <div class="container">
    <div class="row">
      <div class="col"></div>
      <div class="col"></div>
    </div>
    <div class="row">
      <div class="col"></div>
      <div class="col"></div>
    </div>
    <div class="row">
    </div>
  </div>
</body>
```
with this structure you can have multiple stacked sections in the page with only one container class (here as you can see each row will be represent a section for you so you can put your elements inside of it)
so when to use multiple containers in one page ?
first of all you have to know what is a container exactly. container is the wrapper for the bootstrap grid system without container class you can't use rows and columns.
so now you know that the container class is the basic element in bootstrap grid system so now you need to know that container has two types:
- simple container class
- container fluid class 

the container fluid`<div class="container-fluid"></div>` will take the full width of the viewport (the browser width) so if you want to design a full width website you definitely need this as your wrapper 
simple container `<div class="container"></div>` will aligned you elements in the middle of the viewport (browser) and leave some spacing on the right and on the left. you choose this when you want to design Boxed website (just like the old design of our university website)
now the answer for the question:
you use multiple containers in one page when you want show specific section with full width and at the same time you want another section to be boxed
so you will have structure as this example :
```
<body>
  <div class="container">
    <div class="row">
        your columns
    </div>
  </div>
  <div class="container-fluid">
    <div class="row">
      your columns
    </div>
  </div>
</body>
``` 
edit: if you still not understand bootsrtap grid system make this link your first place to look 
https://www.freecodecamp.org/news/learn-the-bootstrap-4-grid-system-in-10-minutes-e83bfae115da/

**Comment by WaledAlaser** [comment]
yes you are right ..
i used a lot of container
i will edit the code with using this way
and i look on the link and i understand all

**Comment by bahaa-kallas** [comment]
Please when you implement these changes reply to this issue with a link for the commit that solved the issue
Like this: 
Solved by commit 0771600161b191413f9b60eec4283ea58dba03f1
How to link commit inside a comment :
just go to commits and press on copy 
![image](https://user-images.githubusercontent.com/53038640/67634426-14eb0f00-f8bc-11e9-80e8-c4d93dd77329.png)
when you paste you will notice a long number showing in the write section 
![image](https://user-images.githubusercontent.com/53038640/67634442-36e49180-f8bc-11e9-95b0-eb0c978591a1.png)

but if you switched to preview it will be translated to a link to the commit 
![image](https://user-images.githubusercontent.com/53038640/67634447-4d8ae880-f8bc-11e9-9b6d-000fedd90844.png)
Why do you need to do this ? 
it just a good practice for now to mention that you solve this issue and provide a link to the commit which contains the issue solution

**Comment by WaledAlaser** [comment]
i can't understand what you mean ? 
and what i should copy from link  ?
you mean when i apply changes on codes i should to put commits?

**Comment by WaledAlaser** [comment]
now i implement this change and solve this issue 
"solved by commit : 4bb651ec7aa09d1cda270545956acdc53e929344 "

and i change alot of things put i will upload the files later because it isn't ready yet
